aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1132,12 +1154,13 @@ module aptos_framework::stake {
             // Automatically renew a validator's lockup for validators that will still be in the validator set in the
             // next epoch.
             let stake_pool = borrow_global_mut<StakePool>(validator_info.addr);
-            if (stake_pool.locked_until_secs <= timestamp::now_seconds()) {
+            let reconfig_start_secs = get_reconfig_start_time_secs();
+            let now_secs = timestamp::now_seconds();
+            if (stake_pool.locked_until_secs <= reconfig_start_secs) {
                 spec {
-                    assume timestamp::spec_now_seconds() + recurring_lockup_duration_secs <= MAX_U64;
+                    assume now_secs + recurring_lockup_duration_secs <= MAX_U64;
```

Comment:
why use now_secs instead of reconfig_start_secs?

Reply:
i'm not sure: imagine the DKG takes so long that `reconfig_start_time + recurring_lockup_duration < now`, i feel we don't want to set the unlock time to be in the past?